core/spec/models/spree/payment_spec.rb

```diff
@@ -17,12 +17,12 @@
   let(:card) { create :credit_card }
 
   let(:payment) do
-    payment = Spree::Payment.new
-    payment.source = card
-    payment.order = order
-    payment.payment_method = gateway
-    payment.amount = 5
-    payment
+    Spree::Payment.create! do |payment|
```

Comment:
I prefer using just `described_class.create` in the spec for the model itself, otherwise you are testing the factory's output at the same time.

We probably shouldn't be using `PaymentCreate` in this file (that should be in the `payment_create_spec.rb` file), I suspect that's here because PaymentCreate replaced an some old payment behaviour.